backend/import/mo_api.py

```diff
@@ -0,0 +1,112 @@
+#
+# Copyright (c) 2018, Magenta ApS
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+import os
+
+from collections import defaultdict
+
+import requests
+
+from cached_property import cached_property
+
+
+DEFAULT_MO_URL = os.environ.get(
+        'MO_URL', 'http://morademo.atlas.magenta.dk/service'
```

Comment:
This should be configurable 🙂 Preferably using a JSON file somewhere for deployment.

Reply:
It _is_ configurable. Just set the environment variable `MO_URL` and that gives you the value. 

Whether or not to use environment variables for this is, of course, a choice.